package.json

```diff
@@ -10,7 +10,7 @@
     "release": "npx bumpp --tag --commit --push && tsx publish.ts"
   },
   "dependencies": {
-    "tsx": "^3.0.0"
+    "tsx": "esbuild-kit/tsx#antfu-patch-1"
```

Comment:
not sure how to install from github. just waiting https://github.com/esbuild-kit/tsx/pull/11 merged & released. :)